Cargo.lock

```diff
@@ -4,9 +4,9 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.21.0"
+version = "0.22.0"
```

Comment:
I think we have a full `cargo update` here, I'll rebase this just on https://github.com/containers/bootc/pull/669